package.json

```diff
@@ -29,6 +29,9 @@
   },
   "author": "Sebastian McKenzie <sebmck@gmail.com>",
   "license": "MIT",
+  "engines": {
+    "node": ">=4"
```

Comment:
Should there be an upper limit placed here? Although it's probably pretty unlikely that a future Node would break this library, we can't know for sure until they're released :)